src/org/parosproxy/paros/core/scanner/AbstractPlugin.java

```diff
@@ -148,7 +159,17 @@ public void init(HttpMessage msg, HostProcess parent) {
         init();
     }
 
-    public abstract void init();
+    /**
+     * Finishes the initialisation of the plugin, subclasses should add any initialisation logic/code to this method.
+     * <p>
+     * Called after a message has been set. By default it does nothing.

```

Comment:
"Set" not "sent", but maybe it should say "Called after the plugin has been initialised with the message being scanned." instead? That might be more clear on when is called and how the message is used.